Update deployment.md (#26296)
TheOdinProject/curriculum: nodeJS/express_and_mongoose/deployment.md

---

> ## Glitch
> 
> - Glitch enables you to effortlessly deploy simple and small Express apps, much like the one you created in the previous lesson, in just a few easy steps.
> - Github import and export support 
> - The best part of this, you don't have to add your credit card *wink wink* info to deploy your apps freely here.

Reviewer: ```suggestion
### Glitch

- Glitch enables you to effortlessly deploy simple and small Express apps in just a few easy steps.
- Github import and export support.
- No credit card needed
```

> **Free Plan**
> 
> - 1000 hours usage per month for each apps.
> - Unlimited static websites that are always on.
> - Full stack apps that will sleep after 5 mins of inactivity but gets back on if a user wants to access it.

Reviewer: ```suggestion
**Free Plan**

- 1000 hours usage per month for each app.
- Unlimited static websites that are always on.
- Full stack apps will be put to sleep after 5 minutes of inactivity.
```

**Links**

> - [Homepage](https://glitch.com/)

Reviewer: ```suggestion
- [Glitch](https://glitch.com/)
```
